apps/system/utils/security_level_cmd.c

```diff
@@ -0,0 +1,131 @@
+ * either express or implied. See the License for the specific
+ * language governing permissions and limitations under the License.
+ *
+ ****************************************************************************/
+
+#include <tinyara/ss_slot_index.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <sys/types.h>
+#include <sys/prctl.h>
+#include <apps/shell/tash.h>
+#include <security/security_ss.h>
+
+/********************************************************************************
+ * Private Functions
+ ********************************************************************************/
+static void show_usage(void) 
+{
+	printf("\nUsage : sec_level [OPTION]...\n");
+	printf("Set your security level high or low, or get the current security level status.\n\n");
+	printf("Options :\n");
+	printf("high		set security level 1(high)\n");
+	printf("		You cannot see some secure information.\n\n");
+	printf("low		set security level 0(low)\n");
+	printf("		You can see all secure information.\n\n");
+	printf("status		show current value of security level\n\n");
+}
+
+static void tash_security_level(int argc, char **args)
+{
+        security_handle hnd;
+	security_data ss_data = {NULL, 0};
+	char ss_path[7];
+	security_error ret;
+
+	if (argc < 2) {
+		printf("Use [--help] option if you want to know how to use.\n");
+		return;
+	}
+
+	/* Show usage */
+	if (!strncmp(args[1], "--help", 7)) {
+		show_usage();
+		return;
+	}
+
+	/* Init security handler */
+	ret = security_init(&hnd);
+        if (ret != SECURITY_OK) {
+		printf("[security_init] error ret : %d\n", ret);
+		return;
+	}
+
+	/* Form the secure storage path name based on the slot index */
+	snprintf(ss_path, 7, "ss/%d", SS_SLOT_INDEX_SECURITY_LEVEL);
+
+	/* Read 'security_level' variable from secure storage */
+	if (!strncmp(args[1], "status", 7)) {
+		ret = ss_read_secure_storage(hnd, ss_path, 0, &ss_data);
+		if (ret != SECURITY_OK) {
+			printf("[ss_read_secure_storage] error ret : %d\n", ret);
+			return;
+		}
+
+		if (!strncmp(ss_data.data, "1", 2)) {
+			printf("security level : high\n");
+		} else if (!strncmp(ss_data.data, "0", 2)) {
+			printf("security level : low\n");
+		} else {
+			printf("Wrong value is written in security_level.\n");
+			printf("[ss_read_secure_storage] security_level data : %s\n", ss_data.data);
+		}
+		
+		return;
+	}
+
+	/* Change security_level according to user input */
+	ss_data.length = 2;
+	if (!strncmp(args[1], "high", 5)) {
+		ss_data.data = "1";
+	} else if (!strncmp(args[1], "low", 4)) {
+		ss_data.data = "0";
+	} else {
+		printf("Invalid argument (%s).\n", args[1]);
+		show_usage();
+		return;
+	}
+	
+        /* Write 'security_level' variable to secure storage  */
+	ret = ss_write_secure_storage(hnd, ss_path, 0, &ss_data);
+        if (ret != SECURITY_OK) {
+		printf("[ss_write_secure_storage] error ret : %d\n", ret);
+		return;
+	} 
+	
+	printf("Security level is set\n");
+
+#ifdef CONFIG_SECURITY_LEVEL
```

Comment:
You already make a dependency with it. In other words, there is no case not defined CONFIG_SECURITY_LEVEL. This `#ifdef` is unnecessary.